### DISCUSSION TITLE
Slash Commands

### DISCUSSION THREAD

**Discussion by BIGDummyHead** [DSharpPlus/DSharpPlus#877] [Q&A] [answered]
Hello, Discord has slash commands now, which I am sure you are aware of. 

This user on GitHub has implemented Slash commands into Discord, this is not a really bad alternative. But I am curious of when or if this group will implement this into a Nuget package. As it is a great implementation of Discord to have these slash commands and it really improves the bots overall look.

Credit:
[DSharpPlus.SlashCommands by Soyvolon](https://github.com/Soyvolon/DSharpPlus.SlashCommands)

**Comment by Emzi0767** [comment]
This is not the only implementation out there, and no, slash commands do not provide much value. As it stands, it's a very low priority item.

**Comment by Naamloos** [comment, marked as answer]
Update: We're considering a Slash Commands extension for the v5.x version of the library.
